tests/integration/transforms/nodejs/simple/Pulumi.yaml

```
@@ -0,0 +1,3 @@
+name: transformations_simple
```

super nitpick (sorry):

```suggestion
name: transforms_simple
```